docs/ignore.md

```diff
@@ -0,0 +1,66 @@
+---
+id: ignore
+title: Ignoring Code
+---
+
+Prettier offers an escape hatch to ignore a block of code from being formatted.
+
+## JavaScript
+
+A JavaScript comment of `// prettier-ignore` will exclude the next node in the abstract syntax tree from formatting.
+
+For example:
+
+```js
+matrix(
+  1, 0, 0,
+  0, 1, 0,
+  0, 0, 1
+)
+
+// prettier-ignore
+matrix(
+  1, 0, 0,
+  0, 1, 0,
+  0, 0, 1
+)
+```
+
+will be transformed to:
+
+```js
+matrix(1, 0, 0, 0, 1, 0, 0, 0, 1);
+
+// prettier-ignore
+matrix(
+  1, 0, 0,
+  0, 1, 0,
+  0, 0, 1
+)
+```
+
+## JSX
+
+```jsx
+<div>
+ {/* prettier-ignore */}
+ <span     ugly  format=''   />
+</div>;
+```
+
+## CSS
+
+```css
+/* prettier-ignore */
+.my    ugly rule
+{
+
+}
+```
+
+## Markdown
+
+```
```

Comment:
` ```markdown` ( ` ```md` will cause `yarn build` failed)

  ![image](https://user-images.githubusercontent.com/8341033/32412309-71e0ccde-c22e-11e7-9d09-89ed44ffb153.png)

  ![image](https://user-images.githubusercontent.com/8341033/32412307-61fb7d64-c22e-11e7-8802-777f61a31497.png)

Reply:
Fixed. I had to remove `vim` from a bunch of places for the same reason. Should investigate because `vim` is supported by Prism.

Reply:
It seems the `prism` in docusaurus is outdated, I didn't see any `vim` in `website/node_modules/docusaurus/lib/core/Prism.js`, and the new `docusaurus@1.0.0-beta` seems replaced it with `highlight.js`.